src/libraries/System.Runtime.Experimental/tests/System.Runtime.Experimental.Tests.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>System.Runtime.Tests</AssemblyName>
```

Comment:
will this workitem clash with System.Runtime on helix?

Reply:
I don't believe so, for the reasons you gave. Whats the easiest way to check if it conflicts in helix?

Reply:
We can look at the workitems of a job and just make sure that the Experimental workitem exist on the list. If we want to dig deeper we can download the payload of System.Runtime.Tests and make sure the right binary is there.

Reply:
What’s the reason for giving this assembly the name System.Runtime.Tests? If we can avoid that it seems easier.